src/Compilers/CSharp/Test/Semantic/Semantics/DelegateTypeTests.cs

```diff
@@ -10766,6 +10766,145 @@ .maxstack 8
+                """
+                .class private auto ansi sealed '<>f__AnonymousDelegate1'
+                    extends [System.Runtime]System.MulticastDelegate
+                {
+                    .custom instance void [System.Runtime]System.Runtime.CompilerServices.CompilerGeneratedAttribute::.ctor() = (
+                        01 00 00 00
+                    )
+                    // Methods
+                    .method public hidebysig specialname rtspecialname 
+                        instance void .ctor (
+                            object 'object',
+                            native int 'method'
+                        ) runtime managed 
+                    {
+                    } // end of method '<>f__AnonymousDelegate1'::.ctor
+                    .method public hidebysig newslot virtual 
+                        instance int32& Invoke (
+                            int32& arg1,
+                            int32& arg2
+                        ) runtime managed 
+                    {
+                        .param [1]
+                            .custom instance void [System.Runtime]System.Diagnostics.CodeAnalysis.UnscopedRefAttribute::.ctor() = (
+                                01 00 00 00
+                            )
+                    } // end of method '<>f__AnonymousDelegate1'::Invoke
+                } // end of class <>f__AnonymousDelegate1
+                """);
+        }
+
+        [ConditionalFact(typeof(CoreClrOnly))]
+        [WorkItem(63565, "https://github.com/dotnet/roslyn/issues/63565")]
+        public void SynthesizedDelegateTypes_UnscopedRefAttribute_02()
+        {
+            var attributeAssemblyName = GetUniqueName();
+            var comp = CreateCompilation(UnscopedRefAttributeDefinition, targetFramework: TargetFramework.Net60, assemblyName: attributeAssemblyName);
+            var refAttribute = comp.EmitToImageReference();
+
+            string sourceA = """
+                using System.Diagnostics.CodeAnalysis;
+                public class A
+                {
+                    public static ref int F([UnscopedRef] ref int i) => ref i;
+                }
+                """;
+
+            comp = CreateCompilation(sourceA, references: new[] { refAttribute }, targetFramework: TargetFramework.Net60);
+            var refA = comp.EmitToImageReference();
+
+            string sourceB = """
+                class Program
+                {
+                    static void Main()
+                    {
+                        int i = 0;
+                        var d = A.F;
+                        d(ref i);
+                    }
+                }
+                """;
+
+            comp = CreateCompilation(sourceB, references: new[] { refA }, targetFramework: TargetFramework.Net60);
+            comp.VerifyEmitDiagnostics(
+                // (6,17): error CS0656: Missing compiler required member 'System.Diagnostics.CodeAnalysis.UnscopedRefAttribute..ctor'
```

Comment:
>System.Diagnostics.CodeAnalysis.UnscopedRefAttribute..ctor [](http://example.com/codeflow?start=75&length=58)

It doesn't look like we are testing scenario when type is present, but constructor is missing. MakeMemberMissing should help.